orca/topology/alerts/extractor.py

```diff
@@ -88,9 +89,13 @@ def _mapping(self):
         return self.__mapping
 
     def map(self, name, labels):
-        mapping = self._mapping.get(name)
+        mapping = self._mapping['plain'].get(name)
         if not mapping:
```

Comment:
For plain mappings, the mapping complexity is O(1), whereas, for regex-based mappings, it jumps to O(n). In a worst-case scenario, we have to traverse the entire mapping dictionary.

I'm just adding the comment for future consideration. For now, the implementation is valid. Let's try to optimize it in the next iteration.